src/main/kotlin/string/calculator/part/Calculator.kt

```diff
@@ -0,0 +1,15 @@
+package string.calculator.part
+
+import string.calculator.domain.Operator
+
+object Calculator {
+    fun calculate(numbers: MutableList<Long>, operators: MutableList<Operator>): Long {
+        var result = numbers.removeFirst()
+
+        for (i in operators.indices) {
+            result = operators[i].operate(result, numbers[i])
+        }
+
+        return result
+    }
```

Comment:
Collection의 확장함수인 `fold`를 활용해서 아래와 같이 변경해볼 수도 있을 것 같아요. 😃
아래와 같이 처리한다면 메서드 파라미터로 가변 리스트를 받을 필요가 없을 것 같습니다. 😉

```suggestion
    fun calculate(numbers: List<Long>, operators: List<Operator>): Long {
        return operators.foldIndexed(numbers.first()) { index, acc, operator ->
            operator.operate(acc, numbers[index + 1])
        }
    }
```